### DISCUSSION TITLE
Task View is very big.

### DISCUSSION THREAD

**Discussion by alainm23** [elementary/tasks#224] [Ideas]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Feature
I understand that you use the same code to create and edit a task but the design is very large and takes up about half the space of the application. I currently use a laptop with a 15.6 inch screen and a resolution of 1366 * 768 (16: 9)

**Describe the solution you'd like**
Make the task view smaller. 

**Existing work**
**macOS Reminders:** Use a simple line for task and notes.
![image](https://user-images.githubusercontent.com/33765137/95601384-fa506e00-0a18-11eb-9448-0bef5ebe08a0.png)

**Todoist:** Use a simple line for task name and actions buttons to select a date, reminder, etc.
![image](https://user-images.githubusercontent.com/33765137/95601521-2ec42a00-0a19-11eb-9b72-e780f6c4824b.png)

**Planner** Use a simple line for task name and actions buttons to select a date, reminder, etc.
![image](https://user-images.githubusercontent.com/33765137/95601751-86fb2c00-0a19-11eb-8a86-a99dd502b1a3.png)

**Additional context**
![image](https://user-images.githubusercontent.com/33765137/95601098-a34a9900-0a18-11eb-8a65-fca700aa14f5.png)

**Comment by danirabbit** [comment]
Yeah the biggest offender here is that the comment area is so large

I like Todois'st/Planner's approach of having image buttons to add additional information like comments, schedules, alarms, etc.

**Comment by alainm23** [comment]
Comments?

![image](https://user-images.githubusercontent.com/33765137/97953199-ad518480-1d97-11eb-841c-f29689c7fcad.png)
